clevis/Sanity/password-generation/test.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+# vim: dict+=/usr/share/beakerlib/dictionary.vim cpt=.,w,b,u,t,i,k
```

Comment:
I personally prefer not adding editor related stuff to scripts

Reply:
This vim modeline is very widely used among QE. It's basically in every test case and test cases are created from the template by command. I'd like to keep it there.